src/main/app/src/utils/isOphOrganisaatio.js

```diff
@@ -0,0 +1,7 @@
+import { OPETUSHALLITUS_ORGANISAATIO_OID } from '../constants';
+
+const isOphOrganisaatio = oid => {
+  return OPETUSHALLITUS_ORGANISAATIO_OID === oid;
```

Comment:
Tässä on Yoda-ehto (vakio ensin). Tavallista järjestystä on käytetty muualla projektissa, joten olisi hyvä tässäkin.

Reply:
Jos minä saisin valita, tämä pitäisi korjata kaikkialle muualle projektiin näin päin. :D